stl/inc/type_traits

```diff
@@ -521,6 +488,19 @@ _INLINE_VAR constexpr bool is_volatile_v<volatile _Ty> = true;
 template <class _Ty>
 struct is_volatile : bool_constant<is_volatile_v<_Ty>> {};
 
+// STRUCT TEMPLATE is_function
+template <class _Ty>
+_INLINE_VAR constexpr bool is_function_v =
+    !is_reference_v<_Ty> && !is_const_v<const _Ty>; // determine whether _Ty is a function

```

Comment:
Could we switch the order of these terms? Not that it makes the slightest difference, but to me it seems the `!is_const_v<const _Ty>` term should come first since it's more significant: lots of types satisfy `!is_reference_v`, but only a few satisfy `!is_const_v<const T>`.